package.json

```diff
@@ -30,7 +30,7 @@
     "typescript": "^4.5.4"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "react-scripts --openssl-legacy-provider start",
```

Comment:
what is this for ?  I think it doesn't work on some OS

nb: if you really need it, maybe avoid to push it here, just use it in your local